docs/governance/03-voting_process/01-linking_mainchain_address.md

```diff
@@ -159,3 +159,73 @@ To unlink an account (mainchain address), perform the following steps:
 
 
 3. Click **Unlink**. The selected mainchain address is unlinked and no longer listed in the **Linked Address** pane.
+
+### Linking a Multisignature Wallet
+#### Generating Signatures for Multisig Wallets
+Before you can link your multisignature wallet you need to first create signatures for each of the signers for the multisignature address. In order to create a signature you will need the Sphere Wallet desktop application. Each signer will have to repeat the steps outlined below. 
+
+1. Open Sphere and login to your account.
+2. On the left hand navigation, select the Wallet that is associated with (a signer of) the multisignature wallet you are trying to link.
+3. You will now be shown a list of addresses associated with that wallet. Select the _handwriting icon_ for the address in question.
+
+<p>
+<img src={require("/img/docs/dao/dao_linking_sphere_icon.png").default} alt="Sphere Handwriting Icon" width="300" height="200" />
+</p>
+
+
+
+4. Next you will be shown a window for you to sign a message. Steps for this part are as follows:
+    - Enter a value into the textbox which is a combination of the **multisignature address** and **EON address** you are trying to link.
+        - Example: If your EON address is _ABC_ and your multisignature address is _XYZ_, then the value you should enter is as follows: _ABCXYZ_.
```

Comment:
It would be more clear if a real example multisignature and EON address are used in the example to confirm what they should look like and the format

Reply:
And also in the screenshot